app/custom_event.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>RUM Custom Event Integ Test</title>
+        <script src="./loader_standard.js"></script>
+        <link
+            rel="icon"
+            type="image/png"
+            href="https://awsmedia.s3.amazonaws.com/favicon.ico"
+        />
+
+        <script>
+            // Common to all test pages
+            function dispatch() {
+                cwr('dispatch');
+            }
+
+            // Record custom event directly using API
+            function recordEvent() {
+                cwr('recordEvent', {
+                    event_type: 'custom_event_api',
+                    event_data: { customEventVersion: 255 }
+                });
+            }
+
+            function recordEmptyEvent() {
+                cwr('recordEvent', {
+                    event_type: 'custom_event_api',
+                    event_data: {}
```

Comment:
will you always have to send an empty object if you dont want to send any data?

Reply:
Yes, that's the idea.